ansible/playbooks/jenkins/worker/create.yml

```
@@ -72,9 +73,10 @@
 #
 
 - hosts:
-    - test-*intel-ubuntu*-x*
+    - test-*hetzner-ubuntu*-x*
```

This could be 
```suggestion
    - test-hetzner-ubuntu*-x*
```
or even better if it was similar to the `benchmarking` role where it was conditional on an inventory variable rather than based on the hostname. But I'm fine with landing the PR as-is and iterating on it afterwards.

---

Agreed. that linux-perf role is currently *only* used in benchmarking so I'll adjust that to be the same as the other benchmarking role.

My original changes were "tread as lightly as possible, things are potentially this way for a reason", and also "get it done before the deadline"